target-platform/jdt_ls.target

```diff
@@ -1,32 +1,40 @@
-</location>
-</locations>
-<targetJRE path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
-</target>
+<?xml version="1.0" encoding="UTF-8"?><?pde version="3.8"?><target name="Java Language Server Target Definition" sequenceNumber="55">
+<locations>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.mockito.mockito-all" version="1.9.5"/>
+<repository location="http://download.eclipse.org/scout/releases/4.0/testing"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="com.google.gson" version="2.2.4.v201311231704"/>
+<unit id="com.google.gson.source" version="2.2.4.v201311231704"/>
+<unit id="org.apache.commons.io" version="2.2.0.v201405211200"/>
+<unit id="org.apache.commons.lang3" version="3.1.0.v201403281430"/>
+<unit id="org.apache.log4j" version="1.2.15.v201012070815"/>
+<repository location="http://download.eclipse.org/tools/orbit/downloads/drops/S20160916162009/repository/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.buildship.feature.group" version="1.0.21.v20161010-1640"/>
+<unit id="org.eclipse.m2e.feature.feature.group" version="1.8.0.20160921-2002"/>
+<repository location="http://download.eclipse.org/releases/oxygen"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.equinox.core.feature.feature.group" version="1.4.0.v20170103-2026"/>
+<unit id="org.eclipse.equinox.core.sdk.feature.group" version="3.13.0.v20170103-2026"/>
+<unit id="org.eclipse.equinox.executable.feature.group" version="3.7.0.v20161219-1356"/>
+<unit id="org.eclipse.equinox.p2.core.feature.feature.group" version="1.4.0.v20161031-1443"/>
+<unit id="org.eclipse.equinox.p2.core.feature.source.feature.group" version="1.4.0.v20161031-1443"/>
+<unit id="org.eclipse.equinox.p2.rcp.feature.feature.group" version="1.3.0.v20161123-1821"/>
+<unit id="org.eclipse.equinox.sdk.feature.group" version="3.13.0.v20170103-2026"/>
+<unit id="org.eclipse.jdt.source.feature.group" version="3.13.0.v20170104-2000"/>
+<unit id="org.eclipse.sdk.feature.group" version="4.7.0.v20170104-2000"/>
+<repository location="http://download.eclipse.org/eclipse/updates/4.7-I-builds"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="com.google.gson" version="2.7.0.v20161205-1708"/>
+<unit id="com.google.gson.source" version="2.7.0.v20161205-1708"/>
+<unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.1.0.v20170105-0805"/>

```

Comment:
I went in that direction because nested jars should be avoided as much as possible. If they could provide us with a "stable" version that we can consume and update weekly this would be similar to consuming weekly I-builds.